backend/index.js

```diff
@@ -31,18 +31,18 @@ app.set('view engine', 'njk')
 app.use(setupBodyParsers())
 app.use(setupHealthChecks())
 app.use(setupWebSecurity())
+app.use(setupRedirects())
+app.use(setupSass())
+app.use(setupStaticContent({ app }))
```

Comment:
This no longer takes **app** as a parameter now that the nunjuck setup has been pulled out.